uktrade/data-hub-frontend
# Add hint text and change placeholder text in Search bar

## Description of change

As part of the accessibility work, the search bar on the home page has been redesigned to help cater to those with low vision abilities. The WCAG guidelines state that when a page is zoomed in to 400%, you should still be able to see everything logically on the page. The current search bar does not meet this guideline as the placeholder text gets cut off at 400%. 

To work around this, hint text has been added to the search bar on the home page. This hint text contains the current placeholder text. A generic placeholder has been added into the search bar. 

## Test instructions

While on the datahub homepage, you should see the search bar with dark grey hint text saying " Search for company, contact, event, investment project or OMIS order". Inside the search bar, you should see generic placeholder text saying " Enter your search term(s)". 

This is to follow the same design that Data Workspace have: 

<img width="1680" alt="Screenshot 2020-09-23 at 17 14 40" src="https://user-images.githubusercontent.com/46787754/94040060-63e93f00-fdc0-11ea-850c-67616989f588.png">

You should also be able to zoom in to 400% and still see all of the hint text and all of the placeholder text.

## Screenshots
### Before

<img width="1208" alt="Screenshot 2020-09-23 at 17 15 52" src="https://user-images.githubusercontent.com/46787754/94040107-72cff180-fdc0-11ea-9096-34b7cf4ab017.png">

### After

<img width="1070" alt="Screenshot 2020-09-23 at 17 16 31" src="https://user-images.githubusercontent.com/46787754/94040177-89764880-fdc0-11ea-887c-c0894caa4f77.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
